src/charm.py

```diff
@@ -117,6 +151,22 @@ def _configure_hubble(self, event):
             except (ManifestClientError, ConnectError):
                 return self._ops_wait_for(event, "Waiting to retry Hubble removal.", exc_info=True)
 
+    def _configure_hubble_metrics(self):
+        values = self.model.config["enable-hubble-metrics"]
+        if values:
```

Comment:
i think we're in the world of "safe to embrace the walrus":

```suggestion
        if values:= self.model.config["enable-hubble-metrics"]
```